src/resources/elements/primeDesignSystem/pinput-numeric/pinput-numeric.scss

```diff
@@ -0,0 +1,19 @@
+@import "../styles";
+
+pinput-numeric {
+  @extend .pInput;
```

Comment:
There should always be a containing element (as I requested elsewhere), and everything in this file should be contained within it.  This ensures good scoping.

Reply:
isn't the component itself a container which also preserves the scope? This is why I used the component tag name as the selector.